src/components/NewRuntimeModal.js

```diff
@@ -504,15 +516,15 @@ export const NewRuntimeModal = withModalDrawer({ width: 675 })(class NewRuntimeM
   }
 
   renderDeleteDiskChoices() {
-    const { deleteDiskSelected, currentPersistentDiskDetails } = this.state
+    const { deleteDiskSelected, currentPersistentDiskDetails, currentRuntimeDetails } = this.state
     return h(Fragment, [
       h(RadioBlock, {
         name: 'delete-persistent-disk',
         labelText: 'Keep persistent disk, delete application configuration and compute profile',
         checked: !deleteDiskSelected,
         onChange: () => this.setState({ deleteDiskSelected: false })
       }, [
-        p(['Please save your analysis data in the directory ', code({ style: { fontWeight: 600 } }, ['/home/jupyter-user/notebooks']), ' to ensure it’s stored on your disk.']),
+        p(['Please save your analysis data in the directory ', code({ style: { fontWeight: 600 } }, [this.getCurrentMountDirectory(currentRuntimeDetails)]), ' to ensure it’s stored on your disk.']),
```

Comment:
If there is no `currentMountDirectory` this sentence seems like it would read a little confusingly: "Please save your analysis data in the directory  to ensure it’s stored on your disk." I think maybe this whole paragraph should be conditional on `getCurrentMountDirectory` returning a value

Reply:
hmm in order for this paragraph to display, there would need to be a PD otherwise we wouldn't display it at all correct? this is in the delete PD flow which we only display for GCE VMs not dataproc clusters since we don't support PDs in dataproc.

Reply:
I tend to agree with what Emily is seeing. While Gabriela is right about the interaction of those conditions, they feel a little distant in the code for it to read clearly.

TOL: I wonder if having the disk mount point included in the `currentPersistentDiskDetails` or `currentRuntimeDetails` would help.

TOL, part 2: It's a little unsettling that the UI has to know what the mount point is instead of being told by leonardo, especially since it's different depending on the app. It would be great if leonardo could provide this information to the UI.